lib/cli/src/fix/fixes/webpack5.ts

```diff
@@ -0,0 +1,122 @@
+import chalk from 'chalk';
+import dedent from 'ts-dedent';
+import semver from '@storybook/semver';
+import { ConfigFile, readConfig, writeConfig } from '@storybook/csf-tools';
+import { Fix } from '../types';
+import { getStorybookInfo } from '../helpers/getStorybookInfo';
+import { PackageJsonWithDepsAndDevDeps } from '../../js-package-manager';
+
+const logger = console;
+
+/**
+ * Is the user using webpack5 in their project?
+ *
+ * If the user is using a version of SB >= 6.3,
+ * prompt them to upgrade to webpack5.
+ *
+ * - Add manager-webpack5 builder-webpack5 as dev dependencies
+ * - Add core.builder = 'webpack5' to main.js
+ * - Add 'webpack5' as a project dependency
+ */
+
+interface Webpack5RunOptions {
+  webpackVersion: string;
+  storybookVersion: string;
+  main: ConfigFile;
+}
+
+interface CheckBuilder {
+  checkWebpack5Builder: (
+    packageJson: PackageJsonWithDepsAndDevDeps
+  ) => Promise<{ storybookVersion: string; main: ConfigFile }>;
+}
+
+export const webpack5: Fix<Webpack5RunOptions> & CheckBuilder = {
+  id: 'webpack5',
+
+  async checkWebpack5Builder(packageJson: PackageJsonWithDepsAndDevDeps) {
+    const { mainConfig, version: storybookVersion } = getStorybookInfo(packageJson);
+    const storybookCoerced = semver.coerce(storybookVersion).version;
+    if (!storybookCoerced) {
+      logger.warn(`Unable to determine storybook version, skipping webpack5 fix.`);
+      return null;
+    }
+
+    if (semver.lte(storybookCoerced, '6.3.0')) {
+      logger.warn(
+        'Detected SB 6.3 or below, please upgrade storybook if you want to use webpack5.'
```

Comment:
To do that, they can just run (or, at least, attempt to run) `npx sb@next upgrade`, yeah? We should tell them that.

Reply:
It would be good to add this info, yes.